### ISSUE TITLE
organize "big" challenges like Pentathlon ourselves

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/Gridcoin-tasks#127]
Advantage would be that we could make the rules!
e.g. steps to minimize bunkering, ...

idea by @grcjamezz

**Comment by Foggyx420** [comment]
Yes organization is a plus and benefits all. There are always challenges before hand and that does hamper the preparation as well.

**Comment by grctest** [comment]
We'll need to have our own statistics tracking across all participation BOINC teams if we do this.
